lib/statsd_middleware.rb

```diff
@@ -30,6 +30,7 @@ class StatsdMiddleware
     0994-edu-benefits
     0996-higher-level-review
     10-10D
+    10-7959a
```

Comment:
Updated this list to include this form ID after 10-10d (may not be strictly necessary, but felt important for completeness sake)

Reply:
@michaelclement We haven't been updating this...hmmm

Reply:
That's true - 10-10d was in there already, so I figured it would make sense to get this one in too, but given that it's for statsd it's probably fine to ignore. 

Want me to remove this change?